tests/nlu/classifiers/test_diet_classifier.py

```diff
@@ -49,6 +52,120 @@
 from rasa.utils.tensorflow.model_data_utils import FeatureArray
 
 
+@pytest.fixture()
+def default_diet_resource() -> Resource:
+    return Resource("DIET")
+
+
+@pytest.fixture()
+def create_diet(
+    default_model_storage: ModelStorage,
+    default_execution_context: ExecutionContext,
+    default_diet_resource: Resource,
+) -> Callable[..., DIETClassifier]:
+    def inner(
+        config: Dict[Text, Any], load: bool = False, finetune: bool = False
+    ) -> DIETClassifier:
+        if load:
+            constructor = DIETClassifier.load
+        else:
+            constructor = DIETClassifier.create
+
+        default_execution_context.is_finetuning = finetune
+        return constructor(
+            config={**DIETClassifier.get_default_config(), **config},
```

Comment:
Think we can drop this, GraphNode should take care of filling in defaults, no?

Reply:
This is not using the `GraphNode`, just the component directly so we have to do it.

Reply:
Oh right, thanks! Would be cool if we had a way to consistently and automatically do that for GraphComponents as well, will add that to the notion list